From 227617a2a6d0302ace027a70a4dd20dfade37ffc Mon Sep 17 00:00:00 2001
From: "tristan.ramseyer" <tristan.ramseyer@stromerbike.com>
Date: Fri, 20 Sep 2024 19:19:11 +0200
Subject: [PATCH] imx6ul ignore speed grading of otp fuses

i.MX 6UL G2 "MCIMX6G2CVM05A*":  528MHz -> 696MHz
i.MX 6ULL Y2 "MCIMX6Y2CVM08AB": 792MHz -> 900MHz

Upstream-Status: Inappropriate [stromer-specific]
Signed-off-by: tristan.ramseyer <tristan.ramseyer@stromerbike.com>
---
 drivers/cpufreq/imx6q-cpufreq.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/cpufreq/imx6q-cpufreq.c b/drivers/cpufreq/imx6q-cpufreq.c
index 0d15fd468a4d..554937c20e32 100644
--- a/drivers/cpufreq/imx6q-cpufreq.c
+++ b/drivers/cpufreq/imx6q-cpufreq.c
@@ -438,6 +438,7 @@ static int imx6ul_opp_check_speed_grading(struct device *dev)
 	val >>= OCOTP_CFG3_SPEED_SHIFT;
 	val &= 0x3;
 
+	/*
 	if (of_machine_is_compatible("fsl,imx6ul"))
 		if (val != OCOTP_CFG3_6UL_SPEED_696MHZ)
 			imx6x_disable_freq_in_opp(dev, 696000000);
@@ -449,6 +450,7 @@ static int imx6ul_opp_check_speed_grading(struct device *dev)
 		if (val != OCOTP_CFG3_6ULL_SPEED_900MHZ)
 			imx6x_disable_freq_in_opp(dev, 900000000);
 	}
+	*/
 
 	return ret;
 }
-- 
2.43.0

